cookbooks/aws-parallelcluster-environment/recipes/config/update_shared_storages.rb

```diff
@@ -37,7 +37,7 @@
     # get efs to unmount
     node.default['cluster']['unmount_efs_shared_dir_array'], node.default['cluster']['unmount_efs_fs_id_array'], = get_efs(UNMOUNT_ACTION)
     # get efs to mount
-    node.default['cluster']['mount_efs_shared_dir_array'], node.default['cluster']['mount_efs_fs_id_array'], node.default['cluster']['mount_efs_encryption_in_transit_array'], node.default['cluster']['mount_efs_iam_authorization_array'] = get_efs(MOUNT_ACTION)
+    node.default['cluster']['mount_efs_shared_dir_array'], node.default['cluster']['mount_efs_fs_id_array'], node.default['cluster']['mount_efs_encryption_in_transit_array'], node.default['cluster']['mount_efs_iam_authorization_array'], node.default['cluster']['mount_efs_access_point_array'] = get_efs(MOUNT_ACTION)
```

Comment:
Were you able to test this update logic?

Reply:
For me it only mounted EFS successfully after providing this code change. Not sure whether "real" updating will work (This is why i set it to unsupported for now).

Reply:
If its set to unsupported, this logic should probably be removed, unless it's required for updates regardless of efs access points?